### ISSUE TITLE
Three Finger Tap behaves differently to Three Finger Click (Magic Trackpad)

### ISSUE THREAD

**Issue by Draccoz** [rxhanson/Middle-Community#46]
Hi, first of all thanks for this app, it helps a lot with the workflow (especially when a windows user comes to work on mac os).
I am a developer and my main IDE is WebStorm. I noticed that middle click works with Jump-to-definition when used with Three Finger Click, but now with Three Finger Tap. A little bit of investigation showed that Tap translates to Option + MiddleClick, while Click does a simple MiddleClick.
Is there a reason to do so? It would be handy to make those two work the same way. Remapping keys in storm works as a workaround, but I would prefer to keep them as they were.
Thanks,
Dan

**Comment by rxhanson** [comment]
Thanks for reporting! Please try v1.7.7 and see if that works properly for you:
https://middleclick.app/downloads/Middle1.7.7.dmg

**Comment by Draccoz** [comment]
Nope, sadly it still works the same. I can provide any video or debugging information you may need, just let me know if there is anything.

**Comment by rxhanson** [comment]
Just a heads up that I haven't forgotten about this one and will take another look when I'm back on a Middle sprint.
